package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "docs-build",
-  "displayName": "Docs Validation",
-  "description": "Local build Docs repository",
+  "description": "Docs Local Build Extension",
```

Comment:
why delete the displayName

Reply:
<img width="306" alt="Screen Shot 2020-02-18 at 4 29 08 PM" src="https://user-images.githubusercontent.com/511355/74717736-debb0c00-526b-11ea-9931-8a97d8f1ef2c.png">

Reply:
what are you trying to say?

Reply:
I guess to be consistent with others - so how about `Docs Local Validation Extension`

Reply:
the others follow the style `docs-xxx`

Reply:
sorry, I meant to say about `description`

for `name`, I guess `displayName` would fallback to `name` if it does not exist, so after this change the `displayName` would be like `docs-build`?

Reply:
yep

Reply:
but I think the display name should be `docs-validation` now, but I don't want to change the id since we may provide more features(like preview) besides validation in the feature.

Reply:
so, are we going to change the display name to something else after having preview?

Reply:
I think so

Reply:
That make sense.